### DISCUSSION TITLE
有没有伪代码相关的模板

### DISCUSSION THREAD

**Discussion by wangdm17** [tuna/thuthesis#618] [Q&A] [answered]
如题，如果想在文章里展示伪代码，应该如何做？模板中好像没有类似的部分。

**Comment by zepinglee** [comment, marked as answer]
使用 `algorithm` 或者 `algorithm2e`。

**Reply by zepinglee** [reply]
需要在示例中添加算法吗？

**Reply by zepinglee** [reply]
需要在示例中添加算法吗？

**Reply by wangdm17** [reply]
感谢，我需要在文章中添加一段伪代码，以说明程序结构，如果示例中可以增加这部分说明，应该可以帮到一些有需求的同学。

**Reply by wenyuhaokikika** [reply]
overleaf上有很多例子吧
